edb/pgsql/compiler/relgen.py

```diff
@@ -71,26 +72,19 @@
 from . import relctx
 
 
+@dataclasses.dataclass(repr=False, eq=False)
```

Comment:
This is a good change but why was ruff complaining?

Reply:
There is a rule for detecting stuff that could be a dataclass: https://docs.astral.sh/ruff/rules/class-as-data-structure/